docs/sources/tempo/operations/monitor/query-io-and-timestamp-distance.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Monitor query I/O and span timestamp distance
```

Comment:
This was a quick stab at creating some doc. If this ends up needing a lot of doc, then let's hide it and iterate on it.